evap/contributor/tests/test_views.py

```diff
@@ -284,3 +286,17 @@ def test_display_request_buttons(self):
         page = self.app.get(self.url, user=self.responsible)
         self.assertEqual(page.body.decode().count("Request changes"), 0)
         self.assertEqual(page.body.decode().count("Request creation of new account"), 2)
+
+
+class TestContributorResultsExportView(WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        result = create_evaluation_with_responsible_and_editor()
+        cls.url = reverse("contributor:export")
+        cls.user = result["responsible"]
+
+    def test_view_downloads_excel_file(self):
```

Comment:
Maybe we can find better name, what do you think of `test_concise_header` or something similar?